### PR TITLE
feat: set canonical header for file

### PR DESCRIPTION
**Pull request by mamico**
File has multiple urls for same content, eg

http://domain/path/foo.pdf/@@download/file
http://domain/path/foo.pdf/@@download/file/foo.pdf
http://domain/path/foo.pdf/@@display-file/file
...

This PR is intended to define a canonical url to avoid duplication and SEO penalties.

see https://developers.google.com/search/docs/crawling-indexing/consolidate-duplicate-urls#rel-canonical-header-method

### CODE REVIEW

**Comment by mamico** [comment]
@jenkins-plone-org please run jobs

File: news/163.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+set canonical header for file. @mamico
```
Comment (davisagli) [inline news/163.feature:1]:
```suggestion
Set canonical header for file. @mamico
```

**Review by davisagli — APPROVED** [review]
TIL about the Link header. Nice idea. LGTM

**Comment by davisagli** [comment]
@jenkins-plone-org please run jobs

**Comment by mamico** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**